Comments.Test/Infrastructure/TestBase.cs

```diff
@@ -314,15 +314,21 @@ protected void AddCommentsAndQuestionsAndAnswers(string sourceURI, string commen
             AddAnswer(questionId, createdByUserId, answerText, status, passedInContext);
         }
 
-        protected void SetupTestDataInDB()
+        protected int SetupTestDataInDB()
```

Comment:
Could we use a helper method instead of having SetupTestDataInDB return a questionId?